### ISSUE TITLE
Feature request: Default support for CAPS LOCK keyboard shortcut

### ISSUE THREAD

**Issue by chirag64** [maximus136/change-string-case#3]
It would be great if you can set a default keyboard shortcut (like `Caps lock`) for either toggling between upper case and lower case or toggling between all the case options cyclically (but only when text is selected of course).

I've always wondered why Desktop Operating systems don't come with this behavior out of the box. This works great on Android with GBoard (highlighting text and tapping Shift-arrow icon will toggle highlighted text between lower case, upper case & title case).

Any thoughts?

**Comment by maximus136** [comment]
Yep, I can try and add keyboard shortcuts to the most-used ones like uppercase and lowercase. Need to explore this though as I am not sure how these keybindings can be implemented. Will take this up in the coming weeks, but feel free to raise a PR if you have any ideas around it.

**Comment by maximus136** [comment]
Added a key-binding for toggling between upper and lower versions of any case, in this [commit](https://github.com/maximus136/change-string-case/commit/33822f1d5d7ced212d8beff418de4f78db887c31).
A single binding for cyclically changing the case is not feasible here considering the number of cases supported here.
Let me know your thoughts, or please close this issue if this works for you.

**Comment by chirag64** [comment]
Just installed v1.1.1 from the marketplace. Doesn't seem to work when I press `Cmd + Caps Lock`. 

![Screen Shot 2019-04-29 at 12 14 08 PM](https://user-images.githubusercontent.com/591424/56910503-b7931400-6a78-11e9-9b34-dfadf5e2d03f.png)

**Comment by maximus136** [comment]
For keybindings to work, you may have to register the commands first, by running any command using the command palette (cmd+shift+p).

**Comment by chirag64** [comment]
That seems to work. But I need to press `Cmd + CAPS LOCK` twice for the shortcut to work. Is that expected?
